src/plugins/git/gitclient.cpp

```diff
@@ -2325,7 +2325,11 @@ GitClient::StatusResult GitClient::gitStatus(const FilePath &workingDirectory, S
         arguments << "--untracked-files=all";
     if (mode & NoSubmodules)
         arguments << "--ignore-submodules=all";
-    arguments << "--porcelain" << "-b";
+
+    for (const auto & option: options) {
+        arguments << option;
+    }
+    arguments << " -- " << workingDirectory.toString();
```

Comment:
Is this extra working directory argument really necessary?

Reply:
I could drop it out. In an older version there was no directory change to `workingDirectory`..